data/data/config.tf

```diff
@@ -32,6 +32,17 @@ EOF
 
 }
 
+variable "masters_schedulable" {
+  type = bool
+
+  default = false
+
+  description = <<EOF
+Whether master nodes are schedulables.
```

Comment:
```suggestion
Whether master nodes are schedulable.
```

Reply:
Oops. Since it's a backport, I'd prefer we keep the patch unchanged and leave the typo there.